tools/verify-all.sh

```diff
@@ -45,6 +49,12 @@ if [[ "${VERIFY_GOVET:-true}" == "true" ]]; then
   cd "${KUBE_ROOT}"
 fi
 
+if [[ "${VERIFY_VENDOR:-true}" == "true" ]]; then
```

Comment:
As far as I can tell VERIFY_VENDOR is not set. That suggests that tools/verify-vendor.sh is not being run as part of verify-all? That would suggest its not being added as part of the pre-submit checks.

Reply:
This set `VERIFY_VENDOR` default to true. It will be skipped if manually set to false. And vendor verification is currently run on presubmit of this PR.

Reply:
@cici37 Would you add an example successful run of verify-all to this PRs description? That should show that verify vendor is being run as expected.

Reply:
Added a link to pow job which runs vendor check in presubmit